### DISCUSSION TITLE
[BUG] Infinity loading when commiting edited file

### DISCUSSION THREAD

**Discussion by MidlajYH** [community/community#40530] [Mobile] [answered]
https://user-images.githubusercontent.com/85442225/205015322-c98c3e75-9fe9-408c-a742-2ac91c03adee.mp4

However, the changes will apply if i close the app and reopen it.

https://user-images.githubusercontent.com/85442225/205015494-135eecdf-9f42-45d7-b520-ddace9541a41.mp4

Device: Poco X3 Pro 
Android Version: 12

Using Github Mobile Latest Beta v1.92.0-beta (10099)

**Comment by jfullstackdev** [comment]
can you try that by logging in your mobile browser, preferably Chrome, and see if you have the same lagging

**Comment by alcere** [comment, marked as answer]
@MidlajYH Thanks for reporting and sorry for the inconvenience! 
Is this something that you can consistently reproduce?

**Reply by alcere** [reply]
The team found the problem and it will be fixed very shortly! Thank you again for the report!

**Reply by MidlajYH** [reply]
> The team found the problem and it will be fixed very shortly! Thank you again for the report!

Glad to hear that. 🚀

**Reply by alcere** [reply]
You should receive an update (Beta v1.92.1) soon!

**Reply by MidlajYH** [reply]
Just updated now, works great 🚀
